### PR TITLE
[PERF-271] SIS Certification Tests: PowerShell Automation

### PR DESCRIPTION
**Pull request by vimayya**

### CODE REVIEW

File: TestRunner.psm1 [hunk 1]
```diff
@@ -545,9 +547,43 @@ function Invoke-TestRunner {
             }
         }
         else{
-            $command = "locust -f $($testSuite)_tests.py -c $clientCount -r $hatchRate --no-web --csv $runnerOutputPath/$testType $runTimeArgument --only-summary"
-            Write-DebugLog "Executing: $command" -LogLevel $logLevel
-            Invoke-Expression -Command $command
+            $outputDir = Resolve-Path $testKitOutputPath
+            Push-Location ./src/edfi-performance-test
+            try {
+                $command = "poetry run python edfi_performance_test --baseUrl $url --key  $key --secret $secret --output $outputDir --testType $testSuite"
```
Comment (stephenfuqua) [inline TestRunner.psm1:553]: It seems that we have PowerShell code to read from config, then pass that to the application. But the application can read the config file on its own, Would it make sense to simplify, compared to the old invocation that passes command line arguments, and just let the application read the .env directly?

**Review by stephenfuqua — COMMENTED** [review]
Can we run this manually on the Azure servers?

File: TestRunner.psm1 (hunk 1 above)
Comment (vimayya) [reply TestRunner.psm1:553]: We need command line arguments for soak test and stress test as they use volume tests with different arguments.

**Comment by vimayya** [comment]
> Can we run this manually on the Azure servers?

Yes. Tests run, I saw some endpoints failing and there are environmental issues to resolve. We could use PERF-272 for that.

**Review by stephenfuqua — APPROVED** [review]

**Pull request merged**